### PR TITLE
add org tags to pipeline&actionAgent log

### PR DESCRIPTION
**Pull request by littlejiancc**
#### What type of this PR

Add one of the following kinds:
/kind polish

#### What this PR does / why we need it:
add org tags to action agent log

#### Which issue(s) this PR fixes:

- Fixes #your-issue_number
- [Erda Cloud Issue Link](https://erda.cloud/erda/dop/projects/387/issues/gantt?filter__urlQuery=eyJpdGVyYXRpb24iOls3NzJdLCJtZW1iZXIiOlsiMTAwMTI2MSJdfQ%3D%3D&id=267283&iterationID=772&pId=0&type=REQUIREMENT)

#### Specified Reviewers:

/assign @sfwn 

#### ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |              |
| 🇨🇳 中文    |              |

#### Need cherry-pick to release versions?

Add comment like `/cherry-pick release/1.0` when this PR is merged.

> For details on the cherry pick process, see the [cherry pick requests](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md#how-to-cherry-pick-a-merged-pr) section under [CONTRIBUTING.md](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: modules/dop/services/autotest_v2/scene.go [hunk 1]
```diff
@@ -604,6 +604,19 @@ func (svc *Service) ExecuteDiceAutotestScene(req apistructs.AutotestExecuteScene
 		})
 	}
 
+	space, err := svc.GetSpace(scene.SpaceID)
+	if err != nil {
+		return nil, err
+	}
+	project, err := svc.bdl.GetProject(uint64(space.ProjectID))
```
Comment (sfwn) [inline modules/dop/services/autotest_v2/scene.go:611]: Is this bundle simple enough?

File: modules/dop/services/autotest_v2/scene.go [hunk 2]
```diff
@@ -604,6 +604,19 @@ func (svc *Service) ExecuteDiceAutotestScene(req apistructs.AutotestExecuteScene
...
+	if err != nil {
+		return nil, err
+	}
+	org, err := svc.bdl.GetOrg(project.OrgID)
```
Comment (sfwn) [inline modules/dop/services/autotest_v2/scene.go:615]: and this.

File: modules/pipeline/pipengine/actionexecutor/plugins/apitest/logic/main.go [hunk 3]
```diff
@@ -59,7 +59,10 @@ func parseConfFromTask(task *spec.PipelineTask) (EnvConfig, error) {
 //   push log to collector
 //   save metadata
 func Do(ctx context.Context, task *spec.PipelineTask) {
-	logger := newLogger().WithContext(context.WithValue(context.Background(), CtxKeyCollectorLogID, task.Extra.UUID))
+	logger := newLogger().
+		WithContext(context.WithValue(context.Background(), CtxKeyCollectorLogID, task.Extra.UUID)).
+		WithField(FieldOrgName, task.Extra.Labels["DICE_ORG_NAME"]).
```
Comment (sfwn) [inline modules/pipeline/pipengine/actionexecutor/plugins/apitest/logic/main.go:64]: uppercase or lowercase ?

File: modules/pipeline/spec/pipeline_extra.go [hunk 4]
```diff
@@ -198,3 +198,13 @@ func (extra *PipelineExtra) GetOrgName() string {
 	}
 	return ""
 }
+
+func (extra *PipelineExtra) GetOrgID() string {
+	if extra.NormalLabels != nil {
```
Comment (sfwn) [inline modules/pipeline/spec/pipeline_extra.go:203]: get from filterLabels and normalLabels by order.
and adjust method to get org name.

**Review by sfwn — CHANGES_REQUESTED** [review]

File: modules/dop/services/autotest_v2/scene.go (hunk 2 above)
Comment (littlejiancc) [reply modules/dop/services/autotest_v2/scene.go:615]: yes

File: modules/dop/services/autotest_v2/scene.go (hunk 1 above)
Comment (littlejiancc) [reply modules/dop/services/autotest_v2/scene.go:611]: Without quota by default

**Review by sfwn — APPROVED** [review]

**Pull request merged**